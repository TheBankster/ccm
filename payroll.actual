{"cpId": 2, "actual": {"SecureBoot": "False", "AntiMalwareCheck": "Stuxnet!!!"}}
{"cpId": 4, "actual": {"RecentlyPatched": "False", "LeastPrivilege": "False"}}
