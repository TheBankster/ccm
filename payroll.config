[settings]
host = localhost
port = 2113

tracing = True
