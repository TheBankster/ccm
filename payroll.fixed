{"cpId": 2, "actual": {"SecureBoot": "False", "AntiMalwareCheck": "Passed"}}
{"cpId": 4, "actual": {"RecentlyPatched": "False", "LeastPrivilege": "True"}}
