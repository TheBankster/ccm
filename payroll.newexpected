{"cpId": 2, "owner": "F648724", "expected": {"SecureBoot": "True", "AntiMalwareCheck": "Passed"}}
{"cpId": 4, "owner": "V077496", "expected": {"RecentlyPatched": "False", "LeastPrivilege": "True"}}
