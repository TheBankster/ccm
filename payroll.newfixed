{"cpId": 2, "actual": {"SecureBoot": "True", "AntiMalwareCheck": "Passed"}}
{"cpId": 4, "actual": {"RecentlyPatched": "False", "LeastPrivilege": "True"}}
