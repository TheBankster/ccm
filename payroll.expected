{"cpId": 1, "owner": "N702766", "expected": {"CSP": "Azure", "SOC3": "False"}}
{"cpId": 2, "owner": "F648724", "expected": {"SecureBoot": "False", "AntiMalwareCheck": "Passed"}}
{"cpId": 3, "owner": "E031494", "expected": {"CodeVersion": 1, "ConfigurationHash": 6528376}}
{"cpId": 4, "owner": "V077496", "expected": {"RecentlyPatched": "False", "LeastPrivilege": "True"}}
